mobile/pubspec.yaml

```diff
@@ -32,7 +32,7 @@ dependencies:
   flutter_svg: ^2.0.9
   package_info_plus: ^8.0.1
   url_launcher: ^6.2.4
-  http: ^0.13.6
+  http: ^1.1.0
```

Comment:
Why is this updated? Can we bump the dependencies in a separate PR? Presumably it should be tested along with other functionality unrelated to file downloads.

Reply:
It is the required version by the background_downloader package